src/corecel/Assert.cc

```diff
@@ -64,33 +49,31 @@ std::string build_debug_error_msg(DebugErrorDetails const& d)
  */
 std::string build_runtime_error_msg(RuntimeErrorDetails const& d)
 {
-    static bool const verbose_message = determine_verbose_message();
+    static bool const verbose_message = [] {
+#if CELERITAS_DEBUG
+        // Always verbose if debug flags are enabled
+        return true;
+#else
+        // Verbose if the CELER_LOG environment variable is defined
+        return !celeritas::getenv("CELER_LOG").empty();
+#endif
+    }();
 
     std::ostringstream msg;
 
-    if (d.which != RuntimeErrorType::validate || verbose_message)
+    msg << "celeritas: " << color_code('R') << to_cstring(d.which)
+        << " error: " << color_code(' ') << d.what;
+
+    if (verbose_message || d.which != RuntimeErrorType::validate
+        || d.what.empty())
     {
-        msg << color_code('W') << d.file;
+        msg << '\n' << color_code('W') << d.file;
```

Comment:
If I read the code correctly, wouldn't the output in case of `d.what.empty() == true` be something like:
```
celeritas: kind_of_error error: 
filename.cxx
```
?

Reply:
It says you've commented on an outdated code? And with the refinements of #1198 here's what the various combinations print as:
<img width="336" alt="Screenshot 2024-04-24 at 12 37 37" src="https://github.com/celeritas-project/celeritas/assets/741229/d9b485bd-ea8b-4117-aab8-f5b70b6e70f5">

Reply:
> It says you've commented on an outdated code?

I got to this already merged PR by way of #1198, so that might be why github complains.